installed.txt

```diff
@@ -5,3 +5,5 @@ canbus-utils
 canbus-utils
```

Comment:
This file should be kept empty. Otherwise initial installation will show some software installed. Whereas, that will not be the case.